File: 2st/Yeonwoo/이어 붙인 수.py
def solution(num_list):
    answer = 0
    even = ''
    odd = ''
    for i in num_list:
        if i%2==1:
            odd += str(i)
        elif i%2==0:
            even += str(i)
    answer = int(even) + int(odd)
    return answer

File: 3st/Juram/programmer/이어 붙인 수.js
function solution(num_list) {
    var odd = "";
    var even = "";
    var toStr = "";
    for(let i=0; i<num_list.length; i++){
        if(num_list[i]%2==0){
            toStr = num_list[i].toString();
            even += toStr;
        }else{
            toStr = num_list[i].toString();
            odd += toStr;  
        }
    }
    return Number(even)+Number(odd);
}